packages/core/src/html/headerProcessor.ts

```diff
@@ -9,25 +9,31 @@ const _ = {
   has,
 };
 
-/*
- * h1 - h6
+/**
+ * Increment the counter for same heading text, and return the id of the heading
+ * if is called from SiteLinkManager, it will directly return the id without adding counter
+ * heading text refers to the h1 - h6 text content
  */
-export function setHeadingId(node: MbNode, config: NodeProcessorConfig) {
+export function processHeadingId(node: MbNode,
```

Comment:
```suggestion
export function processAndReturnHeadingId(node: MbNode,
```
Ok not sure about this name but